src/legacy/ui/public/visualize/loader/pipeline_helpers/__snapshots__/build_pipeline.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`visualize loader pipeline helpers: build pipeline buildPipelineVisFunction handles input_control_vis function 1`] = `"input_control_vis visConfig='{\\"some\\":\\"nested\\",\\"data\\":{\\"here\\":true}}' "`;
 
-exports[`visualize loader pipeline helpers: build pipeline buildPipelineVisFunction handles markdown function 1`] = `"kibana_markdown visConfig='{\\"markdown\\":\\"## hello _markdown_\\",\\"foo\\":\\"bar\\"}' "`;
+exports[`visualize loader pipeline helpers: build pipeline buildPipelineVisFunction handles markdown function 1`] = `"markdownvis '## hello _markdown_' "`;
```

Comment:
The unit test that generates this snapshot should probably be updated to include all of the possible args: https://github.com/elastic/kibana/blob/6285cb76b4d65b8c8c656db962c1badf5db303a3/src/legacy/ui/public/visualize/loader/pipeline_helpers/build_pipeline.test.js#L89-L93 

It also still has `foo: 'bar'` in there, which is getting swallowed